Trio/Sources/Modules/Treatments/TreatmentsStateModel.swift

```diff
@@ -474,14 +475,19 @@ extension Treatments {
                     }
                     await apsManager.enactBolus(amount: maxAmount, isSMB: false, callback: nil)
                 } else {
-                    print("authentication failed")
+                    debug(.bolusState, "Authentication failed")
```

Comment:
From looking at the UnlockManager, it looks like the `unlock` function can never return `false` so this branch will never execute.